crypto/src/hash.rs

```diff
@@ -269,7 +268,6 @@ impl<T> HashOf<T> {
     }
 }
 
-#[cfg(any(feature = "std", feature = "ffi_import"))]
```

Comment:
please look at the tests at the end of this file. Should they remain feature gated?